uaclient/entitlements/base.py

```diff
@@ -28,6 +28,15 @@
 
 
 class UAEntitlement(metaclass=abc.ABCMeta):
+    @property
+    @abc.abstractmethod
+    def help_doc_url(self) -> "Optional[str]":
```

Comment:
I don't think I saw a response as to why this can't just be `help_doc_url = None` on the super-class, only being overriden in sub-classes that need to. (As this has a sensible default value, I don't think we need it to be abstract to _force_ sub-classes to pick a value.)

Reply:
oops, I misunderstood your original comment. Right, we don't need subclasses to define this property. A simple superclass class-attr would do. Correcting that now.